### ISSUE TITLE
IndexError('list index out of range',)

### ISSUE THREAD

**Issue by chrisderwahre** [Fuyukai/OWAPI#231]
## Issue

Request returns index error

## Links that cause the issue

`https://owapi.net/api/v3/u/Dad-12262/blob`

^ returns this index error:

```
{"exc": "IndexError('list index out of range',)", "msg": "please report this!", "error": 500}
```

![screenshot_91](https://user-images.githubusercontent.com/23343464/42113359-5f002b56-7beb-11e8-81af-b95c2f259c59.png)

Greetings, Chris.

**Comment by shane1470** [comment]
I have had this issue as well, but something interesting that I found was that pulling `/stats` doesn't seem to have this issue occur. I also ended up testing this with all the other endpoints as well (`/blob`,  `/achievements`, and `/heroes`), and it appears that they all continue to have this issue however. But for some reason, `/stats` still appears to work just fine. Hopefully this helps!

-Shane
